packages/std/src/errors/backtrace.rs

```diff
@@ -31,6 +40,23 @@ impl Display for BT {
     }
 }
 
+#[cfg(not(feature = "std"))]
+struct Stub;
```

Comment:
Don't we need `impl Printable for Stub`? Maybe we can use `allow(unused)` instead of disabling it to ensure this always compiles

Reply:
Printable has a blanket implementation, so it's implemented already.
Good point with the `allow(unused)`, will do that.